### DISCUSSION TITLE
求助大佬（我这边回复不了，多谢大佬回复）

### DISCUSSION THREAD

**Discussion by yyhack110** [fastfetch-cli/fastfetch#1439] [General]
各位万分抱歉啊，因为我是另外一个项目联系不上作者，没办法才来这里的，我想请教作者@Carter Li 的另外一个库vue3-ace-editor，如何实现自定义关键词高亮的，试了好几种方法，搞了好几天了都不生效，能不能帮忙看看，再次感谢，如下：
![image](https://github.com/user-attachments/assets/c7dc214e-f874-4303-bf54-b7d97c2f50da)

**Comment by CarterLi** [comment]
你拿到了ace的实例，所有操作和vue3-ace-editor已经没关系了。你应该找ace官方去问。
